### ISSUE TITLE
the trained brain doesnot works and shows tfexception says (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.)

### ISSUE THREAD

**Issue by kiir33** [Unity-Technologies/ml-agents#1420]
It gives me the below error when using the trained brain as internal brain and the environment doesnot seems to run.

TFException: NodeDef mentions attr 'output_dtype' not in Op<name=Multinomial; signature=logits:T, num_samples:int32 -> output:int64; attr=seed:int,default=0; attr=seed2:int,default=0; attr=T:type,allowed=[DT_FLOAT, DT_DOUBLE, DT_INT32, DT_INT64, DT_UINT8, DT_INT16, DT_INT8, DT_UINT16, DT_HALF]; is_stateful=true>; NodeDef: multinomial/Multinomial = Multinomial[T=DT_FLOAT, output_dtype=DT_INT64, seed=0, seed2=0](dense_2/MatMul, multinomial/Multinomial/num_samples). (Check whether your GraphDef-interpreting binary is up to date with your GraphDef-generating binary.).
TensorFlow.TFStatus.CheckMaybeRaise (TensorFlow.TFStatus incomingStatus, System.Boolean last) (at <6ed6db22f8874deba74ffe3e566039be>:0)

**Comment by Phantomb** [comment]
I had this issue on one pc and not on the other. Fixed it by pushing the file `Assets/ml-agents/Plugins/Computer/libtensorflow.dll` which had apparently changed from the pc where it did work to the pc where it didn't work.

**Comment by ervteng** [comment]
As suggested, please try re-installing the latest libtensorflow DLL using the Install [TensorFlowSharp guide](https://github.com/Unity-Technologies/ml-agents/blob/master/docs/Basic-Guide.md#setting-up-tensorflowsharp).

**Comment by ervteng** [comment]
Thanks for reaching out to us. Hopefully you were able to resolve your issue.  We are closing this due to inactivity, but if you need additional assistance, feel free to reopen the issue.
